.github/workflows/release.yml

```
@@ -26,6 +26,14 @@ on:
         - 'false'
         type: choice
         required: true
+      tagJava11:
+        description: 'Tag Java 11 variants of images.'
```

```suggestion
        description: 'Tag Java 11 variants of images. Only set True when releasing 1.31.x and later minor versions.'
```
Or per expected release?